dist/LICENSE

```diff
@@ -288,32 +288,4 @@ University, UC Irvine, or Vanderbilt University to appear in their names.
 If you have any suggestions, additions, comments, or questions, please let
 me know.
 
-Douglas C. Schmidt
-
-
----------------------------------------------------------------------------
-The MIT License (http://opensource.org/licenses/MIT)
----------------------------------------------------------------------------
-
-Apache Geode bundles the following files under the MIT License:
-
-  - libxml2 (http://xmlsoft.org),
-    Copyright (C) 1998-2012 Daniel Veillard.  All Rights Reserved.
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+Douglas C. Schmidt
```

Comment:
This line should have a line terminator and show unmodified.

Reply:
Should this file just go away altogether?  The only thing it was referring to was our use of libxml2, so now it conveys essentially no information...

Reply:
It is required.